### PR TITLE
accept STF sha in test pipeline 

### PR DESCRIPTION
**Pull request by xius666**
related to issue #2680

### CODE REVIEW

**Comment by xius666** [comment]
-Grinder STF_OWNER_BRANCH example and its output

1. 

adoptium:master
output from grinder:
Screen Shot 2021-09-23 at 1 20 56 AM

2.

adoptium:2129b191b1111b665758fb755851025e35888a1b
output from grinder:
Screen Shot 2021-09-23 at 7 23 49 PM

3.
adoptium:invalid
output from grinder:
Screen Shot 2021-09-23 at 7 13 04 PM

**Comment by xius666** [comment]
this code also has the change related to the PR #2882

File: system/common.xml [hunk 1]
```diff
@@ -49,17 +49,49 @@
...
+				</exec>
+			</then>
+		</if>
+		
+		<if>
+ 			<equals arg1="${code2}" arg2="128"/>
+			<then>
+				<echo message="This branch does not exist!"/>
+			</then>
+		</if>
```
Comment (llxia) [inline system/common.xml:80]: Could we move the if block inside the above if (after line 71)? We only need to check code2 if we execute line 68.

**Comment by llxia** [comment]
Thanks @xius666 . The PR looks good except for the minor issue above. Also, Please remove the change that is related to PR #2882

File: system/common.xml (hunk 1 above)
Comment (xius666) [reply system/common.xml]: changed it pl review

**Comment by llxia** [comment]
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1485/

**Review by llxia — APPROVED** [review]
Thanks @xius666

**Review by Mesbah-Alam — APPROVED** [review]
LGTM

**Pull request merged**